### DISCUSSION TITLE
Github Copilot menu not shown in Pycharm

### DISCUSSION THREAD

**Discussion by ivanrome** [community/community#7321] [Copilot Conversations] [answered]
Hello,

After installing the plugin in Pycharm, logging and accept terms
![image](https://user-images.githubusercontent.com/13920663/140423870-24b078fc-140d-4286-a7b5-502fdc7f5d7b.png)

the File menu doesn't have the Github copilot entry
![image](https://user-images.githubusercontent.com/13920663/140424111-6992be74-5430-4b1b-800c-c72c11503c11.png)

How can I activate it?

**Comment by martin-marko** [comment, marked as answer]
It's under Tools - GitHub Copilot

**Comment by bryanacido** [comment]
![image](https://user-images.githubusercontent.com/22974567/140450381-50d84469-ee47-4ba5-80b9-eb291fb1b013.png)
